### PR TITLE
Rename: Loopback -> Offline

### PR DESCRIPTION
**Pull request by kphoenix137**
Change Loopback to Offline (visually for end-user) and revise Automap text

Credit goes to @ikonomov for this suggestion

### CODE REVIEW

File: Translations/bg.po [hunk 1]
```diff
@@ -367,8 +367,8 @@ msgstr "Отказ"
 msgid "Client-Server (TCP)"
 msgstr "Клиент-сървър (TCP)"
 
-#: Source/DiabloUI/selconn.cpp:15
-msgid "Loopback"
+#: Source/DiabloUI/selconn.cpp:15 # Needs updated translation
```
Comment (AJenbo) [inline Translations/bg.po:370]: Translations should be marked as fussy, this looks like a manually added comment?

Comment (kphoenix137) [reply Translations/bg.po:370]: Not sure what you mean. Yeah I added those comments manually

Comment (AJenbo) [reply Translations/bg.po:370]: https://til.hashrocket.com/posts/wcmc04nort-fuzzy-translation-key-merging-with-gettext

File: Source/multi.cpp [hunk 2]
```diff
@@ -62,6 +62,7 @@ uint8_t gbDeltaSender;
 bool sgbNetInited;
 uint32_t player_state[MAX_PLRS];
 Uint32 playerInfoTimers[MAX_PLRS];
+bool isLoopback;
```
Comment (AJenbo) [inline Source/multi.cpp:65]: Globals starts with capitals:
```suggestion
bool IsLoopback;
```

Preferably there would be a way to check the network type, but I'm not sure if that is possible and probably a bit hairy to get working. So if you correct this bit I think it's good enough to merge.

Comment (kphoenix137) [reply Source/multi.cpp]: Got it!

**Pull request merged**